src/ImageSharp/PixelFormats/PixelConversionModifiers.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+
+using SixLabors.ImageSharp.ColorSpaces.Companding;
+
+namespace SixLabors.ImageSharp.PixelFormats
+{
+    /// <summary>
+    /// Flags responsible to select additional operations which could be effitiently applied in
+    /// <see cref="PixelOperations{TPixel}.ToVector4(SixLabors.ImageSharp.Configuration,System.ReadOnlySpan{TPixel},System.Span{System.Numerics.Vector4},SixLabors.ImageSharp.PixelFormats.PixelConversionModifiers)"/>
+    /// or
+    /// <see cref="PixelOperations{TPixel}.FromVector4Destructive(SixLabors.ImageSharp.Configuration,System.Span{System.Numerics.Vector4},System.Span{TPixel},SixLabors.ImageSharp.PixelFormats.PixelConversionModifiers)"/>
+    /// knowing the pixel type.
+    /// </summary>
+    [Flags]
+    internal enum PixelConversionModifiers
+    {
+        /// <summary>
+        /// No special operation is selected
+        /// </summary>
+        None = 0,
+
+        /// <summary>
+        /// Select <see cref="IPixel.ToScaledVector4"/> and <see cref="IPixel.FromScaledVector4"/> instead the standard (non scaled) variants.
+        /// </summary>
+        Scale = 1 << 0,
```

Comment:
@JimBobSquarePants I keep thinking whether we should replace *Scaled* / *Scale* with *Normalized* / *Normalize* in our terminology. Seems more precize to me. Or is this wrong thinking?

Reply:
> Or is this wrong thinking?

Actually no. Normalization in our situation is **exactly** what we are doing and I think I prefer it! 

It means breaking `IPixel` for consistency but you have my blessing to do so. I'd really like to ensure we have our naming correct for RC1.